cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -124,7 +124,6 @@ public static int _main(String[] _args) throws Exception {
 
         boolean noKeyAuth = false;
 
-        // TODO perhaps allow mode to be defined by environment variable too (assuming $JENKINS_USER_ID can be used for -user)
```

Comment:
https://github.com/jenkinsci/jenkins/pull/4369#discussion_r362975529

If anything we could introduce something like

```bash
export JENKINS_CLI_OPTS="-http -s https://jenkins/ -auth admin:$(cat token)"
```